src/FixedDataTableRow.js

```diff
@@ -140,6 +145,7 @@ class FixedDataTableRowImpl extends React.Component {
         key="fixed_cells"
         isScrolling={this.props.isScrolling}
         height={this.props.height}
+        cellGroupWrapperHeight={this.props.cellGroupWrapperHeight}
```

Comment:
Why not just do 

height={this.props.height || this.props.cellGroupWrapeprHeight}

It feels strange to have cellGroupWrapperHeight inside of CellGroupWrapper.

Reply:
We can't. this.props.height is used in a place where this.props.cellGroupWrapperHeight is not in FDTCellGroup. It's used for styling cellGroup height, whereas we specifically want to target cellGroupWrapper only.

Reply:
Do you think you could describe this a bit better with a box layout picture?  Something which describes which divs are nested within which and what heights we want as well as why they need to be configurable separately?

Reply:
![cellgroupwrapper](https://user-images.githubusercontent.com/11359452/31512966-b663426a-af5a-11e7-9726-9c415e4044ba.png)
![cellgroup](https://user-images.githubusercontent.com/11359452/31512970-b99295ee-af5a-11e7-9ba8-c3fa3b96c7da.png)

How is that?

Reply:
Maybe, I was thinking something like this (correct me if my sketch butchers anything)
![image](https://user-images.githubusercontent.com/1034455/31513361-26600b6a-af5c-11e7-88af-c26e95ff7d32.png)

@KamranAsif do these visualizations help describe what's going on here?

Reply:
Yeah that makes it much cleaner!

Reply:
Ah, thank you Chris!